rosidl_cmake/package.xml

```diff
@@ -12,6 +12,7 @@
 
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
 
+  <exec_depend>python3-empy</exec_depend>
```

Comment:
Should it? I'm not familiar with the tool itself. I just want it to produce a useful package and know it needs empy to do it.

The semantics of the different dependency keys, especially for packages that are themselves build-time dependencies are not entirely clear to me. Maintainer editing is enabled and if you think this needs to change please feel free to make the change.